asteroidsyncserviced/bluez/bluezclient.cpp

```diff
@@ -19,28 +19,50 @@
 
 #include "bluezclient.h"
 #include "dbus-shared.h"
-#include "../libasteroid/services/common.h"
+#include "services/common.h"
+
+#include "bluez_helper.h"
+#include "freedesktop_objectmanager.h"
+#include "freedesktop_properties.h"
+#include "bluez_adapter1.h"
+#include "bluez_agentmanager1.h"
 
 #include <QDBusConnection>
 #include <QDBusReply>
 #include <QDebug>
+#include <stdexcept>
+
+class BluezClientImpl {
+public:
+    BluezClientImpl();
+    QDBusConnection m_dbus;
+    DBusObjectManagerInterface m_bluezManager;
+private:
+    BluezAdapter1 *m_bluezAdapter = nullptr;
+    FreeDesktopProperties *m_bluezAdapterProperties = nullptr;
```

Comment:
It looks like these two private members are never used, you can remove them then.

Reply:
You are right.  In fact, that also means that we no longer need several files in the bluez directory.